### PR TITLE
feature: Add Adyen Blik to Headless

### PR DESCRIPTION
**Pull request by borisprimer**
[JIRA](https://primerapi.atlassian.net/browse/ACC-4219)

### CODE REVIEW

File: Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift [hunk 1]
```diff
@@ -175,8 +175,12 @@ extension MerchantHeadlessCheckoutAvailablePaymentMethodsViewController: UITable
             break
             #endif
         default:
-            redirectManager = try? PrimerHeadlessUniversalCheckout.NativeUIManager(paymentMethodType: paymentMethodType)
-            try? redirectManager?.showPaymentMethod(intent: sessionIntent)
+            do {
+                redirectManager = try PrimerHeadlessUniversalCheckout.NativeUIManager(paymentMethodType: paymentMethodType)
+                try redirectManager?.showPaymentMethod(intent: sessionIntent)
+            } catch {
+                print("\n\nMERCHANT APP\n\(#function)\nerror: \(error)")
```
Comment (marjaimate) [inline Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift:182]: @BorisNikolic do we need to send analytics data, or logs at this point to Kibana?

**Review by jnewc — COMMENTED** [review]
Please add some tests

File: Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift (hunk 1 above)
Comment (borisprimer) [reply Debug App/Sources/View Controllers/MerchantHeadlessCheckoutAvailablePaymentMethodsViewController.swift:182]: Actually no need, this is the debug app code, the view controller utilising headless implementation as an example.

**Comment by borisprimer** [comment]
> Please add some tests

Will do that, totally forgot to add them 🤦

File: Package.resolved
Comment (NQuinn27) [inline Package.resolved:1]: Would you expect this addition here?

File: Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/PrimerRawOTPDataTokenizationBuilder.swift [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+// swiftlint:disable function_body_length
+
+import Foundation
+
+class PrimerRawOTPDataTokenizationBuilder: PrimerRawDataTokenizationBuilderProtocol {
+
+    var rawData: PrimerRawData? {
+        didSet {
+            if let rawPhoneNumberInput = self.rawData as? PrimerOTPData {
```
Comment (NQuinn27) [inline Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/PrimerRawOTPDataTokenizationBuilder.swift:16]: This variable name is a little misleading

**Review by NQuinn27 — APPROVED** [review]
Nice work, a couple of nits

File: Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/PrimerRawOTPDataTokenizationBuilder.swift (hunk 2 above)
Comment (borisprimer) [reply Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Managers/PrimerRawOTPDataTokenizationBuilder.swift]: Fixed and pushed.

File: Package.resolved
Comment (borisprimer) [reply Package.resolved:1]: Ehm, nope...

Comment (borisprimer) [reply Package.resolved:1]: Removed.

**Review by NQuinn27 — APPROVED** [review]

**Pull request merged**